### PR TITLE
passing through percentage values

### PR DESCRIPTION
**Pull request by bashaus**
Some fixes related to #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "license": "MIT",
   "scripts": {
     "lint": "node_modules/.bin/eslint *.js lib/*.js test/",
-    "test": "npm run lint && mocha --check-leaks"
+    "pretest": "npm run lint",
+    "test": "mocha --check-leaks"
   },
```
Comment (Scrum) [inline package.json:11]: I think it's better to leave them separated, otherwise they can become unreadable

Comment (maltsev) [reply package.json:11]: Not sure I follow you. Did you mean combining them back? Anyway, both options look good to me.

**Comment by maltsev** [comment]
@bashaus thanks! I'll leave #2 open because I think we should implement also percent values rewrite once we find a good way to do it.

File: package.json (hunk 1 above)
Comment (Scrum) [reply package.json:11]: oh sorry I thought it was the other way around)

**Pull request merged**